spec/fabricators/invitation_fabricator.rb

```diff
@@ -0,0 +1,5 @@
+ Fabricator(:invitation) do
+  recipient_name { Faker::Name.name }
+  recipient_email { Faker::Internet.email }
+  message { Faker::Lorem.paragraph }
+ end
```

Comment:
Actually it looks like your `Fabricator` and `end` are the problem because they seem to have a leading space.